.github/workflows/ci.yml

```
@@ -149,10 +149,13 @@ jobs:
         name: Checkout
         uses: actions/checkout@v3
       - name: Install Docker ${{ matrix.engine }}
-        run: |
-          sudo apt-get install curl
-          curl -fsSL https://get.docker.com -o get-docker.sh
-          sudo sh ./get-docker.sh --version ${{ matrix.engine }}
+        uses: crazy-max/ghaction-setup-docker@v2
+        with:
+          daemon-config: |
+            {
+              "debug": true,
+              "version": ${{ matrix.engine }}
+            }
```

I don't think `version` is valid field for daemon config, I think you want to set the the `version` input instead:

```suggestion
        with:
          version: ${{ matrix.engine }}
          daemon-config: |
            {
              "debug": true
            }
```